### PR TITLE
CODE_OF_CONDUCT.md is added, based on other repositories of fluxcd

### LINKED ISSUE #4: Add CODE_OF_CONDUCT.md
Add CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by selinsezi**
Fixes #4 
Adds CODE_OF_CONDUCT.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Instances of abusive, harassing, or otherwise unacceptable behavior in Kubernetes may be reported by contacting the [Kubernetes Code of Conduct Committee](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. For other projects, please contact a CNCF project maintainer or our mediator, Mishi Choudhary <mishi@linux.com>.
+
+This Code of Conduct is adapted from the Contributor Covenant
+(http://contributor-covenant.org), version 1.2.0, available at
+http://contributor-covenant.org/version/1/2/0/
+
+### CNCF Events Code of Conduct
+
+CNCF events are governed by the Linux Foundation [Code of Conduct](https://events.linuxfoundation.org/code-of-conduct/) available on the event page. This is designed to be compatible with the above policy and also includes more details on responding to incidents.
```
Comment (scottrigby) [inline CODE_OF_CONDUCT.md:58]: This file should point to the CNCF CoC, not copy it.

See:
- https://github.com/fluxcd/flux/blob/master/CODE_OF_CONDUCT.md
- https://github.com/fluxcd/toolkit/blob/main/CODE_OF_CONDUCT.md
- https://github.com/fluxcd/helm-controller/blob/main/CODE_OF_CONDUCT.md

And other CNCF projects:
- https://github.com/kubernetes/community/blob/master/code-of-conduct.md
- https://github.com/helm/community/blob/master/code-of-conduct.md


Essentially, this:
```suggestion
# Community Code of Conduct

Flux follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

**Review by scottrigby — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## Flux Community Code of Conduct## Flux Community Code of Conduct
+
+FluxCD Community follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
```
Comment (hiddeco) [inline CODE_OF_CONDUCT.md:4]:
```suggestion

```

(There should only be _one_ trailing white line)

**Review by hiddeco — COMMENTED** [review]
Thanks for making the change as requested! It would be awesome if you could [limit your commit message to a maximum of 50 characters](https://chris.beams.io/posts/git-commit/#seven-rules).

For example:

```
Add `CODE_OF_CONDUCT.md`

Based on other repositories of CNCF projects.
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+## Flux Community Code of Conduct## Flux Community Code of Conduct
```
Comment (scottrigby) [inline CODE_OF_CONDUCT.md:1]: This should not repeat. And should be a H1 (even if if some other Flux repo have as a H2. Those should probably be changed as well).
```suggestion
# Flux Community Code of Conduct
```

**Review by scottrigby — CHANGES_REQUESTED** [review]

**Comment by scottrigby** [comment]
@selinsezi Also I'm not sure if you're aware of the ability to commit review suggestions directly in GitHub. If not, check this out: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

**Comment by dholbach** [comment]
Thanks for your work on this. I merged #20 instead, so I'll close this now.

**Pull request closed without merging**